rpa-bot/1.0/modules/ROOT/edits/configuration.adoc

```diff
@@ -0,0 +1,137 @@
+
+* *Proxy*
++
+The bot and RPA Manager communicate through the HTTP protocol at Port 80. If you
+connect to the internet through a proxy server, enter the proxy address in this field
++
+Format: `+http://<Serveraddress>:<Port>+`, for example: `+http://myproxy.de:881+`.
+
+[[secure-session]]
+== Secure Session
+
+Enable the bot to use Secure Sessions by entering credentials the bot uses to login to the computer and to open a hidden session. If you don't add credentials here, the bot can only use a running desktop session.
+
+image:bot-configurator-secure-session.png[The RPA Bot configurator application showing the Secure Session settings panel, 50%, 50%]
+
+* *Default Secure Session User*
++
+If you want the bot to use Secure Sessions, enter the name of the user the bot uses to login to the computer and to run a hidden session. If the environment of the computer the bot is installed on is embedded in a domain and you want to use a local user, write the user name as follows: `.\username`.
+* *Activate certificate based login on SSL enabled terminal servers*
++
+Enable this option if you are installing the bot on a terminal server that only allows SSL protected RDP connections.
+* *Handle “Always prompt client for password” option*
++
+There is a Group Policy called `fPromptForPassword`. If this policy is enabled, the password has to be entered a second time, when the RDP session is established. If this is the case, enable this option. Also choose the correct form of dialog. On some machines the second password entry is requested by a separate window in form of
+a popup dialog. And on other machines it is an embedded screen like the normal windows login.
+//To determine if the option as to be used and which form of dialog needs to be configured, it is a good idea to execute the steps described in chapter Functional test.
+* *Handle a “Custom LogonMessage”*
++
+It is possible to configure an additional screen which is displayed after the login into an RDP session. It is called Custom Logon Message and normally needs to be acknowledged by pressing a button. If such a message appears when you connect to your system via RDP, enable this option.
+* *Allow usage of SmartCards inside Secure Sessions*
++
+Enables authentication via SmartCard within the Secure Session.
+* *Use the Credential Security Support Provider (CredSSP) for authentication*
++
+Select this option to use the CredSSP for authentication.
+* *Use alternative RDP handling for secure session login (recommended for Windows 10 and Windows Server 2016)*
++
+If the Group Policy _Always prompt for password upon connection_ is enabled, the password must be entered twice when logging into the RDP session. This might prevent the Secure Session from running smoothly, especially under Windows 2010 and Windows Server 2016. If this is the case, activate this option so that the credentials can be entered correctly in the second query.
```

Comment:
```suggestion
If the Group Policy _Always prompt for password upon connection_ is enabled, the password must be entered twice when logging into the RDP session. This might prevent the Secure Session from running smoothly, especially on Windows 2010 and Windows Server 2016. If this is the case, activate this option so that the credentials can be entered correctly in the second query.
```